src/prefect/client/orchestration.py

```diff
@@ -1919,6 +1919,7 @@ async def set_flow_run_state(
         """
         state_create = state.to_state_create()
         state_create.state_details.flow_run_id = flow_run_id
+        state_create.state_details.transition_id = uuid4()
```

Comment:
this will let us use a transition id for every `/set_state` request made via the client